go/vt/servenv/buildinfo.go

```diff
@@ -46,13 +48,14 @@ type versionInfo struct {
 	buildTimePretty string
 	buildGitRev     string
 	buildGitBranch  string
+	buildNumber     int64
 	goVersion       string
 	goOS            string
 	goArch          string
 }
 
 func (v *versionInfo) Print() {
-	fmt.Printf("Version: %s (Git branch '%s') built on %s by %s@%s using %s %s/%s\n", v.buildGitRev, v.buildGitBranch, v.buildTimePretty, v.buildUser, v.buildHost, v.goVersion, v.goOS, v.goArch)
+	fmt.Printf("Version: %s (%d) (Git branch '%s') built on %s by %s@%s using %s %s/%s\n", v.buildGitRev, v.buildNumber, v.buildGitBranch, v.buildTimePretty, v.buildUser, v.buildHost, v.goVersion, v.goOS, v.goArch)
```

Comment:
This change sounds reasonable to me in general.

But:
* I would prefer if we do not include the build number by default.
* if there is some text that makes it clear that it's the Jenkins build e.g. `(Jenkins build %d)` instead of `(%d)`

I expect that the majority of users will not set the BUILD_NUMBER environment variable. For them it will be distracting to see a `(Jenkins build 0)` in the output.